lib/src/components/NxModal/NxModal.scss

```diff
@@ -111,7 +111,7 @@
   }
 
   .nx-tab-list {
-    margin: 1px 1px 0 1px;
+    margin: 1px 1px var(--nx-spacing-6x) 1px;
```

Comment:
This this is repeating a non-trivial value now, I'd say it's better to just set `margin-top`, `margin-left`, and `margin-right` and leave `margin-bottom` un-overridden here instead.

Reply:
> This this is repeating a non-trivial value now, I'd say it's better to just set `margin-top`, `margin-left`, and `margin-right` and leave `margin-bottom` un-overridden here instead.

Thanks, cleaned up here: 39053712add71530ea8243258991ee5e469ea10f